irida_import/irida_import.py

```diff
@@ -767,6 +779,7 @@ def import_to_galaxy(self, json_parameter_file, log, hist_id, token=None,
             samples_dict = json_params_dict['_embedded']['samples']
             email = json_params_dict['_embedded']['user']['email']
             addtohistory = json_params_dict['_embedded']['addtohistory']
+            include_assemblies = json_params_dict['_embedded']['includeAssemblies']
```

Comment:
Since this is a new option older IRIDA's won't send that in their JSON so It would be good to add a test for existence and have a default value of false.

Reply:
Added in 9462fa2dc551cb59c00801d1303c7b8185c5ac1a